tests/engine/adhoc_test.go

```diff
@@ -67,28 +72,50 @@ func runTests(t *testing.T, newRuntimeConfig func() *wazero.RuntimeConfig) {
 	})
 }
 
-func testFibonacci(t *testing.T, newRuntimeConfig func() *wazero.RuntimeConfig) {
+func testHugeStack(t *testing.T, newRuntimeConfig func() *wazero.RuntimeConfig) {
 	// We execute 1000 times in order to ensure the JIT engine is stable under high concurrency
```

Comment:
I think in another PR I will separate out the counccurent testing in conccurent_test.go